test/test_operations.py

```diff
@@ -664,6 +670,20 @@ def test_get_xla_tensor(self):
     self.assertEqual(tx, sx.data.cpu())
 
 
+class TestBinaryCrossEntropyLimitValue(XlaTestCase):
+
+  def test_cross_entropy_loss(self):
+
+    def test_fn(pred, target, offset):
+      lossfn = nn.BCELoss()
+      return lossfn(pred - offset, target)
+
+    pred = torch.tensor(1.0)
+    target = torch.tensor(1.0)
+    for offset in [1, 0, 1e-8, 1e-7]:
```

Comment:
There is no need for the kwargs change above.
Just move the function in the loop and let it capture the current context.